Cargo.lock

```diff
@@ -837,39 +848,39 @@ checksum = "a507befe795404456341dfab10cef66ead4c041f62b8b11bbb92bffe5d0953e0"
 [[package]]
 name = "soroban-env-common"
 version = "0.0.16"
-source = "git+https://github.com/stellar/rs-soroban-env?rev=422ce4110f26097e94835e0a70a4ed8c10b5bc16#422ce4110f26097e94835e0a70a4ed8c10b5bc16"
+source = "git+https://github.com/stellar/rs-soroban-env?rev=49421693412bcd454a769d4caee600ee2c28132e#49421693412bcd454a769d4caee600ee2c28132e"
 dependencies = [
- "crate-git-revision",
+ "crate-git-revision 0.0.4",
  "ethnum",
  "num-derive",
  "num-integer",
  "num-traits",
- "soroban-env-macros 0.0.16 (git+https://github.com/stellar/rs-soroban-env?rev=422ce4110f26097e94835e0a70a4ed8c10b5bc16)",
- "soroban-wasmi",
+ "soroban-env-macros 0.0.16",
+ "soroban-wasmi 0.30.0-soroban (git+https://github.com/stellar/wasmi?rev=1a2bc7f)",
```

Comment:
I notice here is using a short hash `1a2bc7f`, instead of the full hash. (and this matches the error message)
Even in the `before` diff, the soroban-wasmi version's full hash (`1a2bc7f3801c565c2dab22021255a164c05a7f76`) was spelled out. 
Not sure if that's the reason. But it looks suspicious.

Reply:
In my bump wasmi env [PR](https://github.com/stellar/rs-soroban-env/pull/897/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L37), I changed the short hash to full hash, maybe that has to do with the previous version showing up as short here?

Reply:
Yeah it probably has to do with the short hash. I updated prev to use commit in env before the release with the same wasmi version as the release so we should be good.